app/controllers/spree/credit_cards_controller.rb

```diff
@@ -0,0 +1,68 @@
+module Spree
+  class CreditCardsController < BaseController
+
+    before_filter :set_credit_card, only: [:destroy]
+    before_filter :destroy_at_stripe, only: [:destroy]
```

Comment:
related to my comment in enterprises_controller.rb

Reply:
Resolved